README.md

```diff
@@ -23,7 +23,8 @@ For the list of command line flags, see [here](export/src/main/java/com/google/c
 
 ## Stackdriver Monitoring
 
-Both the Import and Export adapter include support for Stackdriver Monitoring. 
+Both the Import and Export adapter include support for Stackdriver Monitoring.
+It is enabled by passing --monitoring_project_id parameter (must be the same project in which the adapter is running)
```

Comment:
Minor comment, I think it reads better as:
"It is enabled by specifying the --monitoring_project_id parameter, which must be the same project in which the adapter is running."

Reply:
Ok